.github/workflows/build.yaml

```diff
@@ -7,22 +7,52 @@ name: Buid iOS app
 on:
   workflow_call:
 
+env:
+  spm_dir: './SPM'
+  cocoapods_dir: './CocoaPods'
+  platform: 'iOS Simulator'
+  device_name: 'iPhone 14'
+  ios_version: '16.2'
+
 # clear out default permissions
 permissions: {}
 
 jobs:
-  build:
-    name: Buid iOS app
+  build-spm:
+    name: Buid iOS app - SPM
     permissions:
       contents: read
     runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: ${{ env.spm_dir }}
     steps:
       - uses: actions/checkout@v3
 
+      - name: Build iOS app
+        run: >
+          xcodebuild
+          -project 'TomTomSDKExamples.xcodeproj'
+          -scheme 'TomTomSDKExamples'
+          -destination 'platform=${{ env.platform }},name=${{ env.device_name }},OS=${{ env.ios_version }}'
+
+  build-cocoapods:
+    name: Buid iOS app - CocoaPods
+    permissions:
+      contents: read
+    runs-on: macos-latest
```

Comment:
Do we need to use the latest macOS?

Reply:
This is not a requirement, but specifying the version would prevent any potential issues that may arise in the future.